### ISSUE TITLE
Enhancement: Add setting to export Passwords

### ISSUE THREAD

**Issue by mathetos** [mathetos/export-users-to-csv#20]
Users want to export passwords and not have to deal with a filter. I'll add a warning to it as well.

**Comment by orangutangle** [comment]
How would this feature even be possible? As I understand it, WordPress stores the passwords encrypted with a one-way hash. So to export the passwords, you'd have to reverse the encryption, which in theory should take millions of years.

**Comment by mathetos** [comment]
@orangutangle -- it would be exported encrypted, so that when it's imported via another plugin or other method it would remain the same password in the new site. It's often requested via the Support Forums.

**Comment by orangutangle** [comment]
But wouldn't the salt be different on a different machine?

**Comment by mathetos** [comment]
Yes, that's correct. But the user could ensure they are identical first before importing. Right now, I have a filter that allows passwords to be added to the export, and users have said they want it exported as a setting, so I'm mostly responding to user feedback on this one.
